lib/src/unix/tls.rs

```diff
@@ -0,0 +1,183 @@
+use std::io;
```

Comment:
This module is copied from `hyper-sync-rustls` to workaround [`TlsStream` takes `HttpStream` only as underlying stream impl](https://github.com/SergioBenitez/hyper-sync-rustls/blob/914e770d2697beade61e3a5019e78bcaaf07e8fe/src/lib.rs#L24-L27) now, I'd like to get rid of this and make `hyper_sync_rustls::TlsStream` generic over the underlying stream.